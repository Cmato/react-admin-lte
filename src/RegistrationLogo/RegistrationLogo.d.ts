import * as React from 'react';

export interface RegistrationLogoProps {
}

declare const RegistrationLogo: React.ComponentType<RegistrationLogoProps>;

export default RegistrationLogo;
