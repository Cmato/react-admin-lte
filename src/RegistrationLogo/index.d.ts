export { default } from './RegistrationLogo';
export * from './RegistrationLogo';
