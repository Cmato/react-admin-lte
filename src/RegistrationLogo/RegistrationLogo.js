import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

const RegistrationLogo = ({
  children,
  className: classNameProp,
  ...other
}) => {
  const className = classNames({
    'registration-logo': true
  }, classNameProp);

  return (
    <div className={className} {...other}>
      {children}
    </div>
  );
};

RegistrationLogo.propTypes = {
  /**
   * The content of the component.
   */
  children: PropTypes.node.isRequired,
  /**
   * @ignore
   */
  className: PropTypes.string,
};

export default RegistrationLogo;
