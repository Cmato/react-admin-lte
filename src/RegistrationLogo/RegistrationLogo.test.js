import React from 'react';
import { shallow } from 'enzyme';
import RegistrationLogo from './RegistrationLogo';

it('renders default', () => {
  expect(shallow(<RegistrationLogo>Mine!</RegistrationLogo>))
    .toMatchSnapshot();
});
