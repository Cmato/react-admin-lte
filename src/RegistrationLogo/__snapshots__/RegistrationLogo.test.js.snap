// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default 1`] = `
<div
  className="registration-logo"
>
  Mine!
</div>
`;
